contracts/seed/SeedFactory.sol

```diff
@@ -20,20 +20,19 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 import "./Seed.sol";
 import "../utils/CloneFactory.sol";
 
-
 /**
  * @title primeDAO Seed Factory
  * @dev   Enable primeDAO governance to create new Seed contracts.
  */
 contract SeedFactory is CloneFactory {
     using SafeMath for uint256;
 
-    Avatar    public avatar;
-    Seed      public parent;
-    bool      public initialized;
+    Avatar public avatar;
+    Seed public parent;
+    bool public initialized;
 
-    uint32  public constant PPM               = 1000000;   // parts per million
-    uint256 public constant PPM100            = 100000000; // ppm * 100
+    // uint32  public constant PPM               = 1000000;   // parts per million
```

Comment:
if we decided not to use `PPM` let's delete comments

Reply:
Sure, Making them.